editor/src/core/shared/redux-devtools.ts

```diff
@@ -58,19 +58,15 @@ maybeDevTools?.subscribe((message) => {
   }
 })
 
-const ActionsToOmit: Array<EditorAction['action']> = ['UPDATE_PREVIEW_CONNECTED', 'LOAD']
-const ActionsWithPayload: Array<EditorAction['action']> = [
-  'CREATE_INTERACTION_SESSION',
-  'UPDATE_INTERACTION_SESSION',
-]
-
 let lastDispatchedStore: SanitizedState
 
 const PlaceholderMessage = '<<SANITIZED_FROM_DEVTOOLS>>'
 
-function simplifiedMetadata(elementMetadata: Partial<ElementInstanceMetadata>) {
+function simplifiedMetadata(elementMetadata: ElementInstanceMetadata) {
   return {
+    elementPath: EP.toString(elementMetadata.elementPath),
```

Comment:
this is still a very small subset of the metadata, if anyone needs more stuff, they can add it here, on demand. but trying to log the full metadata chokes Devtools